cfme/fixtures/service_fixtures.py

```diff
@@ -26,26 +26,33 @@ def catalog():
 
 
 @pytest.fixture(scope="function")
-def catalog_item(provider, provisioning, vm_name, dialog, catalog):
-    template, host, datastore, iso_file, catalog_item_type, vlan = map(provisioning.get,
-        ('template', 'host', 'datastore', 'iso_file', 'catalog_item_type', 'vlan'))
-    item_name = dialog.label
-    provisioning_data = dict(
-        vm_name=vm_name,
-        host_name={'name': [host]},
-        datastore_name={'name': [datastore]},
-        vlan=vlan
-    )
+def catalog_item(provider, provisioning, console_template, vm_name, dialog, catalog):
+    def _catalog_item(use_console_template=False):
+        # This nested function allows to pass value for use_console_template attribute
+        # which is derived from request.param in the order_catalog_item_in_ops_ui fixture
+        template, host, datastore, iso_file, catalog_item_type, vlan = map(provisioning.get,
+            ('template', 'host', 'datastore', 'iso_file', 'catalog_item_type', 'vlan'))
+        if use_console_template:
+            logger.info("Console template name : {}".format(console_template.name))
+            template = console_template.name
+        item_name = dialog.label
+        provisioning_data = dict(
+            vm_name=vm_name,
+            host_name={'name': [host]},
+            datastore_name={'name': [datastore]},
+            vlan=vlan
+        )
 
-    if provider.type == 'rhevm':
-        provisioning_data['provision_type'] = 'Native Clone'
-    elif provider.type == 'virtualcenter':
-        provisioning_data['provision_type'] = 'VMware'
-    catalog_item = CatalogItem(item_type=catalog_item_type, name=item_name,
-                  description="my catalog", display_in=True, catalog=catalog,
-                  dialog=dialog, catalog_name=template,
-                  provider=provider, prov_data=provisioning_data)
-    return catalog_item
+        if provider.type == 'rhevm':
+            provisioning_data['provision_type'] = 'Native Clone'
+        elif provider.type == 'virtualcenter':
+            provisioning_data['provision_type'] = 'VMware'
+        catalog_item = CatalogItem(item_type=catalog_item_type, name=item_name,
+                      description="my catalog", display_in=True, catalog=catalog,
+                      dialog=dialog, catalog_name=template,
+                      provider=provider, prov_data=provisioning_data)
+        return catalog_item
+    return _catalog_item
```

Comment:
REQUIRED: The ``catalog_item`` fixture is used in many services tests and this change of behavior will break these tests.
Better approach would be to create function like ``create_catalog_item(provider, provisioning, vm_name, dialog, catalog, console_template=None)`` and make the ``catalog_item`` fixture use that function.
In the ``order_catalog_item_in_ops_ui`` fixture you wouldn't use the ``catalog_item`` fixture, but call the ``create_catalog_item`` function and pass value for ``console_template`` parameter.